### PR TITLE
Update CSS token per new design system theming capability

### PR DESCRIPTION
**Pull request by fajar-apri-alaska**
# Alaska Airlines Pull Request

**Fixes:** #153 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

**Comment by fajar-apri-alaska** [comment]
The  `npm ci` is failing because some auro components have conflicting peer dependency for `@aurodesignsystem/webcorestylesheets` package. Should I hold of this until those components are updated?

File: demo/index.html [hunk 1]
```diff
@@ -23,6 +23,7 @@
       href="https://cdn.jsdelivr.net/npm/prismjs@1.24.1/themes/prism.css"
     />
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css">
```
Comment (blackfalcon) [inline demo/index.html:26]: Please update to mirror this code 

https://github.com/AlaskaAirlines/auro-avatar/blob/main/demo/index.html#L12-L17

File: src/style.scss [hunk 2]
```diff
@@ -3,12 +3,14 @@
 
 /* stylelint-disable max-nesting-depth */
 
-@import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
+@import './../node_modules/@aurodesignsystem/webcorestylesheets/src/utilityClasses/displayProperties';
+
+// Support for fallback values
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables.scss';
```
Comment (blackfalcon) [inline src/style.scss:9]:
```suggestion
@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables;
```

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Please be sure to update all package dependencies and update repo to support our Node support policy https://auro.alaskaair.com/node-support#gsc.tab=0

**Comment by fajar-apri-alaska** [comment]
> Please be sure to update all package dependencies and update repo to support our Node support policy https://auro.alaskaair.com/node-support#gsc.tab=0

Does this means that we will need to work on changes based on [node 18 updates](https://github.com/AlaskaAirlines/auro-flight/pull/64) for this component also? Sure, I could do that in a new issue ticket.

File: package.json [hunk 3]
```diff
@@ -16,64 +16,64 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=18.15.0"
```
Comment (jason-capsule42) [inline package.json:19]: The `.github/workflows/testPublish.tml` file needs to be configured to execute the testing on the same node versions we are flagging here in the package.json.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (fajar-apri-alaska) [reply package.json:19]: I see, I will follow [this](https://github.com/AlaskaAirlines/auro-avatar/pull/45/files#diff-3358abb4a7abaf5862dcf8a68e95ee8431a7adf98d0d4c576444fff439dd665aR18) change as reference.

Comment (blackfalcon) [reply package.json:19]: Please reference this page in the docSite. https://auro.alaskaair.com/node-support. Please update the `settings.yml` file too.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**